packages/mask/src/components/shared/SelectRecipients/SelectRecipients.tsx

```diff
@@ -8,6 +14,8 @@ import { useTwitterIdByWalletSearch } from './useTwitterIdByWalletSearch'
 import { isValidAddress } from '@masknet/web3-shared-evm'
 import { useI18N } from '../../../utils'
 import { cloneDeep, uniqBy } from 'lodash-unified'
+import Services from '../../../extension/service'
+import type { ProfileRecord } from '../../../../background/database/persona/type'
```

Comment:
Please do not use this type. It is marked as internal (only can be used in the background). If you need to modify the profile, please use another representation.